scheduler/CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file
  
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.51.7] - 2021-03-11
+### Added
+- Make cook pods ignore a tenured node taint, from @scrosby
+- Add support to python jobclient for mea culpa retries, from @CGe0516
+
```

Comment:
We're missing this:

https://github.com/twosigma/Cook/commit/ec5cc23753ff689089acd9e277a15402743b74a6